teammapper-backend/src/map/services/maps.service.spec.ts

```diff
@@ -152,6 +208,33 @@ describe('MapsController', () => {
       expect(await nodesRepo.findOne({ where: { id: node.id } })).not.toBeNull()
     })
 
+    it('deletes a map which has outdated nodes and outdated lastAccess', async () => {
```

Comment:
also add a test (if not given, can't find it right now): Outdated modifedAt, recent nodes => do not delete. This is how its working right now.

Reply:
This is covered by `it('does not delete a map that contains a recent node')`, where lastAccessed is set to 2019 but the node is set to today using `new Date()`.